demo_scripts/AKSDeployment/README.md

```diff
@@ -0,0 +1,314 @@
+Welcome to this tutorial where we will take you step by step in creating an AKS Application with a custom domain that is secured via https. This tutorial assumes you are logged into Azure CLI already and have selected a subscription to use with the CLI. It also assumes that you have helm installed (Instructions ca be found here https://helm.sh/docs/intro/install/). If you have not done this already. Press b and hit ctl c to exit the program.
+
+To Login to Az CLI and select a subscription 
+'az login' followed by 'az account list --output table' and 'az account set --subscription "name of subscription to use"'
+
+To Install Az CLI
+If you need to install Azure CLI run the following command - curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+
+
+Assuming the pre requisites are met press enter to proceed
+
+## Define Command Line Variables 
+Most of these variables should be set to a smart default. However, if you want to change them
+press b and run the command export VARIABLE_NAME="new variable value"
+
+```
+echo $RESOURCE_GROUP_NAME
+echo $RESOURCE_LOCATION
+echo $AKS_CLUSTER_NAME
+echo $PUBLIC_IP_NAME
+echo $VNET_NAME
+echo $SUBNET_NAME
+echo $APPLICATION_GATEWAY_NAME
+```
+
+For the following variables, unless you manually added them in the env.json you will be asked to provide an input
+
+The custom domain must be unique and fit the pattern: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$
+For example mycooldomain 
+
+Do not add any capitalization or .com
+```
+echo $CUSTOM_DOMAIN_NAME
+```
+
+For the email address any enter a valid email. I.e sarajane@gmail.com
+```
+echo $SSL_EMAIL_ADDRESS
+```
+
+## Create A Resource Group
+The first step is to create a resource group. You can do this with the following command
+
+'az group create --name $RESOURCE_GROUP_NAME --location $RESOURCE_LOCATION'
```

Comment:
It doesn't make much sense to have the command in the text like this. It's already there in line 46. This is duplicating content. When running in SimDem if you run in "learn" mode then the command is output for the user to type in. If running in "tutorial" mode then it's added for the user.

This comment goes for all duplicated content. I know we are not trying for perfection here, but we should be testing as it is intended to be used. If you want the user to see the command before executing then use the "learn" mode.

Reply:
Happy to remove the before code blocks if we think it is a better user experience. As a user for the first time, I thought it was a beneficial to know what I was going to run before the command executed. While you get this with learn, I didn't want to type the command or type 'a' every time to execute, I wanted it to just go automatically for me when I hit enter.

Reply:
The design goal is for the content to work as a document and as an interactive tutorial. In this form it doesn't work as a document because there is duplicated information.

If you want to test if users prefer to see the command or not then test some using "tutorial" mode where they will only see it when they execute it and some using "learn" where they will be shown it and will need to either type it in or hit 'a' for auto.

Both use cases are already supported.